### PR TITLE
Adds 4km hurricane with SE and RK4 options

### PR DESCRIPTION
**Pull request by pwolfram**
Adds 4km hurricane with SE and RK4 options using 2km hurricane as template,
but distinguishing forward mode into split explicit (SE) and 4th order Runge Kutta (RK4)
options.

### CODE REVIEW

**Comment by pwolfram** [comment]
# Testing

Results are relatively smooth with `ssh_gradient` as well as between SE and RK4 without wetting and drying for the lower-resolution case, but results obviously aren't as good as at more appropriate higher resolutions (e.g., 2km and 1km).  The case includes self attraction and loading.

**Comment by pwolfram** [comment]
@mark-petersen, you are welcome to review if you'd like but since this is COMPASS my plan is to just merge once @sbrus89 reviews.  This should lead into the next PR for the flooding via wetting and drying.

**Comment by pwolfram** [comment]
@sbrus89 , cleaned up location of station day per our conversation.  This is cleaner now and prevents duplication of files as discussed.

File: testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4/build_mesh/run.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+"""
+This script was generated as part of a driver_script file by the
+setup_testcases.py script.
+"""
+import sys
+import os
```
[the remark this answers is not in the document]
Comment (pwolfram) [reply testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4/build_mesh/run.py]: Removed now, thanks

File: testing_and_setup/compass/ocean/hurricane/USDEQU120at30cr10rr2/sandy/config_init.xml [hunk 2]
```diff
@@ -4,10 +4,10 @@
 	<add_link source="../../build_mesh/culled_mesh/culled_graph.info" dest="graph.info"/>
 	<add_link source_path="initial_condition_database" source="sandy_wnd10m.nc" dest="wnd10m.nc"/>
 	<add_link source_path="initial_condition_database" source="sandy_prmsl.nc" dest="prmsl.nc"/>
-	<add_link source_path="script_configuration_dir" source="interpolate_time_varying_forcing.py" dest="interpolate_time_varying_forcing.py"/>
-	<add_link source_path="script_configuration_dir" source="create_pointstats_file.py" dest="create_pointstats_file.py"/>
-	<add_link source_path="script_test_dir" source="NOAA-COOPS_stations.txt" dest="NOAA-COOPS_stations.txt"/>
-	<add_link source_path="script_test_dir" source="USGS_stations.txt" dest="USGS_stations.txt"/>
+  <add_link source_path="script_configuration_dir" source="scripts/interpolate_time_varying_forcing.py" dest="interpolate_time_varying_forcing.py"/>
```
Comment (sbrus89) [inline testing_and_setup/compass/ocean/hurricane/USDEQU120at30cr10rr2/sandy/config_init.xml:7]: Tab issue

File: testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4/sandy/config_init.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<config case="init">
+	<add_link source="../../build_mesh/culled_mesh/culled_mesh.nc" dest="mesh.nc"/>
+	<add_link source="../../build_mesh/culled_mesh/culled_graph.info" dest="graph.info"/>
+	<add_link source_path="initial_condition_database" source="sandy_wnd10m.nc" dest="wnd10m.nc"/>
+	<add_link source_path="initial_condition_database" source="sandy_prmsl.nc" dest="prmsl.nc"/>
+  <add_link source_path="script_configuration_dir" source="scripts/interpolate_time_varying_forcing.py" dest="interpolate_time_varying_forcing.py"/>
```
Comment (sbrus89) [inline testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4/sandy/config_init.xml:7]: Tab issue

File: testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4/sandy/config_init.xml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+	<add_link source_path="script_configuration_dir" source="scripts/create_pointstats_file.py" dest="create_pointstats_file.py"/>
+  <add_link source_path="script_configuration_dir" source="sandy_stations/NOAA-COOPS_stations.txt" dest="NOAA-COOPS_stations.txt"/>
```
Comment (sbrus89) [inline testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4/sandy/config_init.xml:9]: Tab issue

File: testing_and_setup/compass/ocean/hurricane/USDEQU60at15cr5rr1/sandy/config_init.xml [hunk 5]
```diff
@@ -4,10 +4,10 @@
 	<add_link source="../../build_mesh/culled_mesh/culled_graph.info" dest="graph.info"/>
 	<add_link source_path="initial_condition_database" source="sandy_wnd10m.nc" dest="wnd10m.nc"/>
 	<add_link source_path="initial_condition_database" source="sandy_prmsl.nc" dest="prmsl.nc"/>
-	<add_link source_path="script_configuration_dir" source="interpolate_time_varying_forcing.py" dest="interpolate_time_varying_forcing.py"/>
-	<add_link source_path="script_configuration_dir" source="create_pointstats_file.py" dest="create_pointstats_file.py"/>
-	<add_link source_path="script_test_dir" source="NOAA-COOPS_stations.txt" dest="NOAA-COOPS_stations.txt"/>
-	<add_link source_path="script_test_dir" source="USGS_stations.txt" dest="USGS_stations.txt"/>
+  <add_link source_path="script_configuration_dir" source="scripts/interpolate_time_varying_forcing.py" dest="interpolate_time_varying_forcing.py"/>
```
Comment (sbrus89) [inline testing_and_setup/compass/ocean/hurricane/USDEQU60at15cr5rr1/sandy/config_init.xml:7]: Tab issue

**Comment by sbrus89** [comment]
I was able to reproduce @pwolfram's results and compared them to the 2km case:

**Comment by sbrus89** [comment]
I was able to reproduce @pwolfram's results and compared them to the 2km case:

**Comment by pwolfram** [comment]
Thanks @sbrus89 !

**Pull request merged**